chronicle/release/releasers/github/summarizer.go

```diff
@@ -100,78 +117,124 @@ func (s *Summarizer) LastRelease() (*release.Release, error) {
 	return nil, fmt.Errorf("unable to find latest release")
 }
 
-//nolint:funlen
 func (s *Summarizer) Changes(sinceRef, untilRef string) ([]change.Change, error) {
-	var changes []change.Change
-	var err error
-
-	var includeStart, includeEnd bool
+	scope, err := s.getChangeScope(sinceRef, untilRef)
+	if err != nil {
+		return nil, err
+	}
 
-	var sinceTag *git.Tag
-	sinceHash := sinceRef
-	if sinceRef != "" {
-		sinceTag, err = s.git.SearchForTag(sinceRef)
-		if err != nil {
-			return nil, err
-		}
-		includeStart = false
-	} else {
-		includeStart = true
+	if scope == nil {
+		return nil, fmt.Errorf("unable to find start and end of changes: %w", err)
 	}
 
-	var sinceTime *time.Time
-	if sinceTag != nil {
-		sinceRelease, err := fetchRelease(s.userName, s.repoName, sinceTag.Name)
-		if err != nil {
-			return nil, fmt.Errorf("unable to fetch release %q: %w", sinceTag.Name, err)
-		}
-		sinceTime = &sinceRelease.Date
+	return s.changes(*scope)
+}
+
+func (s *Summarizer) getChangeScope(sinceRef, untilRef string) (*changeScope, error) {
+	sinceTag, sinceRef, includeStart, sinceTime, err := s.getSince(sinceRef)
+	if err != nil {
+		return nil, err
 	}
 
 	var untilTag *git.Tag
-	untilHash := untilRef
+	var untilTime *time.Time
 	if untilRef != "" {
 		untilTag, err = s.git.SearchForTag(untilRef)
 		if err != nil {
 			return nil, err
 		}
-		includeEnd = false
+		untilTime = &untilTag.Timestamp
 	} else {
-		untilHash, err = s.git.HeadTagOrCommit()
+		untilRef, err = s.git.HeadTagOrCommit()
 		if err != nil {
 			return nil, err
 		}
-		includeEnd = true
 	}
 
 	var includeCommits []string
 	if s.config.ConsiderPRMergeCommits {
```

Comment:
I don't understand what this config does.

Reply:
looks like this option never was added to the documentation, I can update that in a separate PR (here's the PR that put this feature in https://github.com/anchore/chronicle/pull/40 ) . In short, when enabled the commits that comprise the release can affect which PRs get included into the release notes.